src/theme/GlobalStyle.ts

```
@@ -1,5 +1,8 @@
 import {css} from '@emotion/react';
 
+import {Theme} from './theme.type';
```

ESLint

[error] 'Theme' is defined but never used.

(@typescript-eslint/no-unused-vars)